src/runtime_src/core/tools/common/ReportDynamicRegion.cpp

```diff
@@ -131,6 +131,7 @@ populate_cus(const xrt_core::device *device)
         ptCu.put( "name",			layout->m_ip_data[i].m_name);
         ptCu.put( "base_address",		boost::str(boost::format("0x%x") % base_addr));
         ptCu.put( "usage",			usage);
+        ptCu.put( "type",			enum_to_str(cu_type::PL));

```

Comment:
What is the type representing here?

Reply:
kernel type PL or PS. populate_cus_new() had this change but this function populate_cus doesn't have it. So, added it here also. On windows XRT, populate_cus() function is getting invoked, and no node called "type" error is coming without this change.

Reply:
why cant we use same function in both linux and windows?  populate_cus_new should be called here as well right?

Reply:
kds_mode is 0 on windows, hence populate_cus() is being invoked.

Reply:
Got it. Thanks Raj for the clarification.